modulefiles/wflow_cheyenne.lua

```diff
@@ -10,11 +10,12 @@ load("ncarenv")
 append_path("MODULEPATH","/glade/p/ral/jntp/UFS_SRW_app/modules")
 load("rocoto")
 
-load(pathJoin("conda", os.getenv("conda_ver") or "latest"))
+prepend_path("MODULEPATH","/glade/work/epicufsrt/contrib/miniconda3/modulefiles")
+load(pathJoin("miniconda3", os.getenv("miniconda3_ver") or "4.12.0"))
 
 if mode() == "load" then
    LmodMsgRaw([===[Please do the following to activate conda:
-       > conda activate /glade/p/ral/jntp/UFS_SRW_app/conda/regional_workflow
+       > conda activate regional_workflow
```

Comment:
Good to see Cheyenne is similar to other systems in this regard noq. I think you would need to do the same in `ush/load_modules_wflow.sh` lines 63-69 
```Bash
-if [[ "${machine}" == 'cheyenne' ]]; then
-    conda activate /glade/p/ral/jntp/UFS_SRW_app/conda/regional_workflow
-else
-    if [ ! -z $(command -v conda) ]; then
-        conda activate regional_workflow
-    fi
+if [ ! -z $(command -v conda) ]; then
+    conda activate regional_workflow
 fi
```